src/tests/unit/tests/reports/components/report-sections/__snapshots__/footer-text.test.tsx.snap

```diff
@@ -8,7 +8,8 @@ exports[`FooterText renders 1`] = `
   , a tool that helps debug and find accessibility issues earlier on
    
   environmentName
-  . Get more information & download this tool at 
+  . Get more information & download this tool at
+   
```

Comment:
This is an odd looking change: mind checking to see if there's any visible change?

Reply:
There's no visible change in the report, it just added a newline in front of the space in the snapshot.